spec/requests/questionings_spec.rb

```diff
@@ -52,4 +52,71 @@
       end
     end
   end
+
+  describe "condition_form_data" do
+
+    let(:form) { create(:form, :published, question_types: %w(integer text integer text)) }
+    let(:qing) { form.questionings[2] }
+
+    context "without ref_qing_id" do
+      it "returns json with ref qing id options, no operator options, and no value options" do
+        expected_ref_qing_options = form.questionings.select{|q| q.id < qing.id }.map{ |q| { code: q.question.code, rank: q.full_dotted_rank, id: q.id } }
```

Comment:
Styleguide calls for space after `map`, `select` etc., before block {.

Reply:
Selecting questionings by comparing IDs won't work after we switch to UUIDS. I think you can just do: `form.c[0..1].map { |q| ... }`.

`form.c` is a shortcut alias for `form.sorted_children`, good for use in specs, especially when you have nested groups like `form.c[2].c[1].c[3]`